pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -759,21 +762,30 @@ def send_notification_email(self, advisories: Dict[str, int], jira_link: str):
 @click_coroutine
 async def prepare_release(runtime: Runtime, group: str, assembly: str, name: Optional[str], date: str,
                           package_owner: Optional[str], nightlies: Tuple[str, ...], default_advisories: bool, include_shipped: bool):
-    # parse environment variables for credentials
-    jira_token = os.environ.get("JIRA_TOKEN")
-    if not runtime.dry_run and not jira_token:
-        raise ValueError("JIRA_TOKEN environment variable is not set")
-    # start pipeline
-    pipeline = PrepareReleasePipeline(
-        runtime=runtime,
-        group=group,
-        assembly=assembly,
-        name=name,
-        date=date,
-        nightlies=nightlies,
-        package_owner=package_owner,
-        jira_token=jira_token,
-        default_advisories=default_advisories,
-        include_shipped=include_shipped,
-    )
-    await pipeline.run()
+    slack_client = runtime.new_slack_client()
+    slack_client.bind_channel(assembly)
```

Comment:
Needs to be bind to `group` instead of `assembly` (which could be something like `rc.1`).